src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -617,7 +619,7 @@ public synchronized void updateEntity(DbEntity entity) throws DatabaseEngineExce
                 }
                 if (!toRemove.isEmpty()) {
                     if (properties.allowColumnDrop()) {
-                        dropColumn(entity, toRemove.toArray(new String[toRemove.size()]));
+                        dropColumn(entity, toRemove.toArray(new String[0]));
```

Comment:
Doing `new String[0]`  means that `toArray` method will allocate an array with the right size, right?